packages/@aws-cdk/aws-events-targets/lib/sqs.ts

```diff
@@ -40,6 +40,9 @@ export interface SqsQueueProps {
 export class SqsQueue implements events.IRuleTarget {
 
   constructor(public readonly queue: sqs.IQueue, private readonly props: SqsQueueProps = {}) {
+    if (typeof props.messageGroupId !== 'undefined' && !queue.fifo) {
```

Comment:
Not sure what this comment means against this line of code.

In general, it looks like we can a FIFO queue can be imported via its ARN like any other SQS queue. See `queue.ts` file in this change to see changes related importing a FIFO queue.

Reply:
Yeah, I guess it might work. It's just a countdown to the moment where someone tries to import a FIFO queue from an ARN in a  Token (which we can't inspect  for a `.fifo` suffix), and it will fail, and they'll file a bug report.

Reply:
Is there a case where someone will encounter that? 

Either a user imports an SQS queue that "physically exists", in which case, no token in the ARN; 
or uses an SQS queue created within the CDK app, in which case, they shouldn't be importing.

Reply:
CFN parameters are a case. Users are encouraged not to do that, and it's unlikely, but they might.

I'm okay putting it off however.